frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,347 @@
+import MnemonicDialog from "src/components/mnemonic/MnemonicDialog";
+import PasswordInput from "src/components/password/PasswordInput";
+import SettingsHeader from "src/components/SettingsHeader";
+import { Alert, AlertDescription, AlertTitle } from "src/components/ui/alert";
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "src/components/ui/alert-dialog";
+import { Badge } from "src/components/ui/badge";
+import { Button } from "src/components/ui/button";
+
+import { Label } from "src/components/ui/label";
+import { LoadingButton } from "src/components/ui/loading-button";
+import { Separator } from "src/components/ui/separator";
+import { useToast } from "src/components/ui/use-toast";
+import { UpgradeDialog } from "src/components/UpgradeDialog";
+import { useAlbyMe } from "src/hooks/useAlbyMe";
+import { useInfo } from "src/hooks/useInfo";
+import { useMigrateLDKStorage } from "src/hooks/useMigrateLDKStorage";
+import { InfoResponse, MnemonicResponse } from "src/types";
+import { request } from "src/utils/request";
+
+export default function Backup() {
+  const { toast } = useToast();
+  const { data: info, hasMnemonic } = useInfo();
+  const { data: me } = useAlbyMe();
+  const [unlockPassword, setUnlockPassword] = useState("");
+  const [decryptedMnemonic, setDecryptedMnemonic] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  const navigate = useNavigate();
+
+  const onSubmitPassword = async (e: React.FormEvent) => {
+    e.preventDefault();
+    try {
+      setLoading(true);
+      const result = await request<MnemonicResponse>("/api/mnemonic", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ unlockPassword }),
+      });
+
+      setDecryptedMnemonic(result?.mnemonic ?? "");
+      setIsDialogOpen(true);
+    } catch (error) {
+      toast({
+        title: "Incorrect password",
+        description: "Failed to decrypt mnemonic.",
+        variant: "destructive",
+      });
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <>
+      <SettingsHeader
+        title="Backup"
+        description={
+          <>
+            <span className="text-muted-foreground">
+              Backup your wallet recovery phrase and channel states. These
+              backups are for disaster recovery only. To migrate your node,
+              please use the migration tool.
+            </span>
+            <a
+              href="https://guides.getalby.com/user-guide/alby-account-and-browser-extension/alby-hub/backups-and-recover"
+              target="_blank"
+              rel="noreferrer noopener"
+              className="text-foreground underline"
+            >
+              {" "}
```

Comment:
![image](https://github.com/user-attachments/assets/3a2b772b-e06f-4a3d-8c81-633c1f4d344f)

please can you pay attention to detail regarding links and spacing